bitnami/rabbitmq/templates/plugins-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.rabbitmq.communityPlugins -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "rabbitmq.fullname" . }}-plugins
+  labels: {{- include "rabbitmq.labels" . | nindent 4 }}
+data:
+  install-plugins.sh: |
+    #!/bin/bash
+    echo "==> Plugin installation"
+    echo $RABBITMQ_PLUGINS_DIR
+    pluginPath=/opt/bitnami/rabbitmq/plugins
+    downloadPath=/tmp/rabbitmq-plugins
+    mkdir $downloadPath
+    {{- $totalPlugins := len .Values.rabbitmq.communityPlugins }}
+    echo "Total plugins defined in chart installation: {{ $totalPlugins }}"
+    {{- range $i, $plugin := .Values.rabbitmq.communityPlugins }}
+    echo "Installing plugin {{ add $i 1 }} out of {{ $totalPlugins }}: {{ $plugin.name }}"
+    curl -X GET "{{ $plugin.url }}" -L -o $downloadPath/{{ $plugin.name }}.zip
+    unzip $downloadPath/{{ $plugin.name }}.zip -d $downloadPath
```

Comment:
Is not possible to unzip the plugins directly in the plugins directory? Doing that we wil save to move them later

Reply:
sounds like a better approach, indeed